<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcul mental</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_Pret">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Prêt à démarrer, paramètres OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Error">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_StoResult">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>67</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>StoResult</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_StoIteration">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>67</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>StoIteration</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Numbres">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>256</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>194</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_NbAdd">
       <property name="text">
        <string>Nombre d'additions :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_NbAdd"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>246</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_Range">
       <property name="text">
        <string>Range des additions :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_RangeFrom"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_RangeTo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>127</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Total">
       <property name="text">
        <string>Saisir le total :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_StoPrevResult">
       <property name="text">
        <string>StoPrevResult</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Total"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Resultat">
       <property name="text">
        <string>label_Resultat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
